test/ShapeCrawler.Tests.Unit/Helpers/EnvironmentChecks.cs

```diff
@@ -0,0 +1,30 @@
+namespace ShapeCrawler.Tests.Unit.Helpers;
+
+public static class EnvironmentChecks
+{
+    public static bool IsGhostscriptInstalled()
```

Comment:
EPS images are supported if the source system has ghostscript so this checks if it is installed. Later the test checks this and marks it as inconclusive if it is. This allows us to check another of the possible exceptions thrown by the library.

Reply:
I think we should handle this differently, in the production code:

```C#
AddPicture(Stream image)
//...
catch (MagickDelegateErrorException ex) when (ex.Message.StartsWith("FailedToExecuteCommand `\"gswin64c.exe\""))
{
    throw new SCException(
        "It seems the stream is the EPS file format that requires gswin64c.exe which is not installed on your system.");
}
//...
```

Reply:
I think we should handle this differently, in the production code:

```C#
AddPicture(Stream image)
//...
catch (MagickDelegateErrorException ex) when (ex.Message.StartsWith("FailedToExecuteCommand `\"gswin64c.exe\""))
{
    throw new SCException(
        "It seems the stream is the EPS file format that requires gswin64c.exe which is not installed on your system.");
}
//...
```

Reply:
Sorry, this is one example of a format supported if GhostScript is installed. The whole list can be found here https://imagemagick.org/script/formats.php. I wonder if we should just wrap the Magick.Net Exceptions as they are quite descriptive themselves?

Reply:
> this is one example of a format supported if GhostScript is installed.

Then the message can be more general, e.g. "It seems the stream is an image format that requires GhostScript which is not installed on your system."

> I wonder if we should just wrap the Magick.Net Exceptions as they are quite descriptive themselves?

Currently, it is a convention that caught exceptions must be wrapped in `SCException`. One of the reasons is that the client application developer can more quickly discover that the problem is in the ShapeCrawler, and we _maybe_ can help because we at the compile time understood the source problem.

You can use constructor `SCException(string message, Exception innerException)` to additionally pass Magick.Net Exception.

Reply:
Done. The check has to be broader as there are differences between the Ghostscript errors depending on the environment.